spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -65,13 +65,16 @@
       expect(response).to redirect_to idv_phone_errors_failure_url
     end
 
-    it 'shows phone form if async process times out' do
+    it 'shows phone form if async process times out and allows successful resubmission' do
       # setting the document capture session to a nonexistent uuid will trigger async
       # timed_out behavior
       subject.idv_session.idv_phone_step_document_capture_session_uuid = 'abc123'
 
       get :new
       expect(response).to render_template :new
+      put :create, params: { idv_phone_form: { phone: good_phone } }
```

Comment:
can/should we put an expectation about rendering a flash message too here?

Reply:
we don't at the moment, so it would fail.

I could add a generic flash message about an internal error and then add the expectation?

Reply:
Yeah I'd add something generic for now, so we have some sort of UI feedback this is happening, we can clean up the copy with input from UX soon

Reply:
added in aac0c27